vinu.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: vinugupta
+pageUrl: vinugupta.github.io
+timestamp: TODAYS-2017-28-03
+---
```

Comment:
@vinugupta you'll need to remove `TODAYS-` and add a line break after line 5. That should allow your tests to pass 🎉